lib/api/controllers/authController.js

```diff
@@ -163,10 +197,121 @@ class AuthController {
   /**
    * List authentication strategies
    *
-   * @returns {Promise}
+   * @returns {Promise.<string[]>}
    */
   getStrategies() {
-    return this.kuzzle.pluginsManager.listStrategies();
+    return Bluebird.resolve(this.kuzzle.pluginsManager.listStrategies());
+  }
+
+  /**
+   * @param {Request} request
+   * @returns {Promise.<Object>}
+   */
+  createMyCredentials(request) {
+    let
+      createMethod,
+      validateMethod;
+
+    assertIsConnected(this.kuzzle, request);
+    assertHasStrategy(request);
+    assertIsStrategyRegistered(this.kuzzle, request);
+    assertHasBody(request);
+
+
+    createMethod = this.kuzzle.pluginsManager.getStrategyMethod(request.input.args.strategy, 'create');
```

Comment:
Generally we try to make Kuzzle "plugin fault tolerent". Here, if the method "create" in the current strategy doesn't exist, what happens?
I guess we have to test methods returned by `getStrategyMethod` and reject if the method doesn't exit. What do you think?
(same comment for others methods)